app/models/media.rb

```diff
@@ -289,6 +290,7 @@ def archive_if_conditions_are_met(options, id, cache)
       cache.read(id, :json).dig('archives').blank? ||
       # if the user adds a new  or changes the archiver, and the cache exists only for the old archiver it refreshes the cache
       options&.dig(:archivers) != cache.read(id, :json)['archives'].keys.join
+        Rails.logger.info level: 'INFO', message: '[Archiver] Archiving new URL', url: self.url
```

Comment:
Fix: I think it's more consistent if we move this line to before this one: https://github.com/meedan/pender/blob/develop/app/models/concerns/media_archiver.rb#L19. In that case, we can also archive the archiver name (e.g., `archiver: name`).

But since you're there, can you please fix that indentation? :)

Reply:
Great suggestion. This is done too.